waiter/src/waiter/scheduler.clj

```diff
@@ -513,7 +513,7 @@
                          (if service-id
                            (let [scheduler-state (service-id->state scheduler service-id)
                                  health-check-context (get current-state service-id)]
-                             (async/>! response-chan (merge scheduler-state health-check-context)))
+                             (async/>! response-chan (merge {} scheduler-state health-check-context)))
```

Comment:
Would we be safer with an integration test that ensured Waiter didn't crash if we passed an invalid service id?

Reply:
Will add.

Reply:
Done.